portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBAdmin.testcase

```diff
@@ -484,9 +484,7 @@ definition {
 	@description = "This is a test for LPS-120917. It checks that XSS cannot be executed from the KB template name."
 	@priority = "3"
 	test CannotStoreXSSInTemplateName {
-		KBAdmin.openKBAdmin(siteURLKey = "guest");
-
-		KBTemplate.addCP(
+		JSONKnowledgeBase.addkBTemplate(
```

Comment:
@chaowang-123 please try to search for existing JSON macros to use first before using the ones in the UI. If there isn't one that exists feel free to create one (and/or ask for help to create one). Thanks!

Reply:
@austinchiang ok, got it, thanks!